docker-compose-deploy.yml

```diff
@@ -0,0 +1,24 @@
+version: "3.7"
+
+services:
+  app:
+    build:
+      context: .
```

Comment:
#context of our build will be our local directory. so we will be using Dockerfile in root